Code/CSharpException/Triangle.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace CSharpException
+{
+    public class Triangle
+    {
+        public Triangle(double sideA, double sideB, double sideC) 
+        {
+            if (sideA <= 0 || sideB <= 0 || sideC <= 0)
+            {
+                throw new ArgumentException("Side value cannot be equal to or less than zero");
+            }
+            if (sideA + sideB <= sideC || sideA + sideC <= sideB || sideB + sideC <= sideA)
+            {
+                throw new ArgumentException("The sum of two sides must be greater than the third side. Please enter a valid values of the triangle sides.");
+            }
+            SideA = sideA;
+            SideB = sideB;
+            SideC = sideC;
+        }
```

Comment:
add spaces between constructor and all properties.

Reply:
done